src/components/Filters/CheckboxOption.tsx

```diff
@@ -78,7 +79,7 @@ export function CheckboxOption(props: CheckboxOptionProps): JSX.Element | null {
     resultsCount
   } = props;
   const cssClasses = useComposedCssClasses(builtInCssClasses, props.customCssClasses);
-  const optionId = useId();
+  const [optionId] = useId();
```

Comment:
out of curiosity, what else does this `useId` method return besides the id?

Reply:
looks like it returns a string array

Reply:
what would the other strings in the array be used for? are we sure we want the first one? also, are we sure the array won't be empty?

Reply:
i took a look at the code and it seems that it always returns one element in the array if no value is being passed into useId() as we are currently doing so i think we should be fine here.